kovri.md

```diff
@@ -1,64 +1,68 @@
+
+Trusted VPNs are also susceptible to trivial bandwidth correlation; making it easy for 3-letter agencies to de-anonymize her traffic.
+VPNs also adhere to local laws in order to stay in business; so Alice always runs the risk of having her user data given to a government whose policies she doesn't agree with.
+Trusted VPNs also don't protect Bob from Alice knowing who Bob is.
+
+It's all so complicated! Fortunately, Alice and Bob have a better option for protecting their privacy. Their option is Kovri.
 
-In a sentence, Kovri is a lightweight router compatible with cryptocurrencies and built for the I2P network.
-I2P stands for Invisible Internet Protocol.
-I2P is a layer of the internet that allows for all connections in the network to be private.
-In contrast with regular internet routing or TOR's onion routing, I2P utilizes what it refers to as garlic routing.
-Instead of connecting with participants directly or through TOR nodes, such as going directly to a website, all information is encrypted and relayed through other I2P routers in the network.
+Kovri is a free, decentralized, anonymity technology developed by Monero.
+Currently based on the Invisible Internet Project's open specifications, Kovri uses both garlic encryption and garlic routing to create a private, protected overlay-network across the internet.
+This overlay-network provides Alice and Bob with the ability to effectively hide their geographical location and internet IP address.
+Essentially, Kovri *covers* an application's internet traffic to make it anonymous within the network.
+A lightweight and security-focused router, Kovri is fully compatible with the I2P network.
 
-Since I2P routers know little or no information about the data they are sending, where it originated, or its destination, all information in this network is private and anonymous.
-If Alice sends her transaction with Kovri, nodes that log Monero traffic will no longer get her real IP address, rendering these passive surveillance tactics useless.
+If Alice sends her transaction with Kovri, nodes that log Monero traffic will no longer see her internet IP address, rendering passive surveillance tactics close to useless.
 
 Now let’s imagine a different scenario.
-Suppose Alice has an internet service provider that does not want her to use Monero, who send her a warning saying so.
-Alice can optionally send all the Monero sync data, not just the transaction broadcast, through I2P.
+Suppose Alice has an internet service provider that does not want her to use Monero, and send her a warning to cease and desist.
+Alice can optionally send all the Monero sync data, not just the transaction broadcast, through Kovri.
 Now, her Monero connections are hidden from her ISP.
 This improves Monero's censorship resistance, since other people no longer know who is running a Monero node and demand that they be shut down.
```

Comment:
"This improves Monero's censorship resistance, since other people no longer know who is running a Monero node, and others can not demand that they be shut down."

Reply:
Agreed.